gbpservice/nfp/test/unit/configurator/agent/lb_test_agent.py

```diff
@@ -0,0 +1,346 @@
+import unittest
+import mock
+
```

Comment:
For this file, apart from the comments provided here, please refer the review comments provided for firewall as well.